### PR TITLE
front end task #4

### PR DESCRIPTION
**Pull request by jvberdec**
addition input from users, info retrieval on views.py

### CODE REVIEW

**Comment by jvberdec** [comment]
Front end task #4

File: cards/forms.py [hunk 1]
```diff
@@ -1,9 +1,16 @@
 from django import forms
 
+CREDIT_CHOICES = [('excellent', 'Excellent'), ('good', 'Good'), ('average', 'Average'), ('bad', 'Bad')]
+ANNUAL_CHOICES = [('yes', 'Yes'), ('no', 'No')]
+BANK_CHOICES   = [('chase', 'Chase'), ('citi', 'Citibank'), ('amex', 'American Express'), ('capital one', 'Capital One'), ('wells fargo','Wells Fargo'), ('bank of america', 'Bank of America'), ('all', 'All of the Above')]
+
```
Comment (natharrow) [inline cards/forms.py:6]: Great! Looks like we'll be able to easily updated the list of banks in case we add more data.

File: cards/forms.py [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
     groceries = forms.DecimalField(label='Groceries', min_value=0)
     dining = forms.DecimalField(label='Dining out', min_value=0)
     gas = forms.DecimalField(label='Gas', min_value=0)
     travels = forms.DecimalField(label='Travels', min_value=0)
-    etc = forms.DecimalField(label='Everything Else', min_value=0)
\ No newline at end of file
+    etc = forms.DecimalField(label='Everything Else', min_value=0)
+    credit_score = forms.CharField(label='What is your credit score?', widget= forms.Select(choices= CREDIT_CHOICES))
+    annual= forms.ChoiceField(label='Are you only interested in cards with no annual fee?', widget= forms.RadioSelect, choices= ANNUAL_CHOICES)
+    # banks = forms.ModelMultipleChoiceField(label= 'Show cards from the following banks:', widget= forms.CheckboxSelectMultiple, choices= BANK_CHOICES)
```
Comment (natharrow) [inline cards/forms.py]: All field types will have to be changed to integers. This is to simplify computations and make the form nicer and simpler.

**Comment by natharrow** [comment]
> We'll have to update views.py to take into account the additional information we are asking for from the forms (credit score, annual fee, banks). We'll also have to remember to remove the test prints in views.py, once the feature is complete.

I will remove the comments since they were for completed features that were tested multiple times and probably won't be changed.

**Pull request merged**